internal/ui/todo-edit-modal.go

```
@@ -384,3 +400,150 @@
+	for i, tag := range ts.availableTags {
+		cursor := " "
+		if ts.focused && ts.cursor == i {
+			cursor = ">"
+		}
+
+		checked := "[ ]"
+		if ts.IsSelected(tag.Name) {
+			checked = "[x]"
+		}
+
+		sb.WriteString(fmt.Sprintf("%s %s %s\n", cursor, checked, tag))
```

golangci-lint (1.64.8)

printf: fmt.Sprintf format %s has arg tag of wrong type *github.com/martijnspitter/tui-todo/internal/models.Tag

(govet)